parmed/amber/parameters.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, *filenames):
     #===================================================
 
     @classmethod
-    def from_leaprc(cls, fname):
+    def from_leaprc(cls, fname, ions=True, solvents=True):
```

Comment:
I'm not sure how I feel about this.  It's satisfying a particular use-case that is specific to the way in which we're currently thinking about converting the Amber parameter files to OpenMM XMLs.  It seems like this functionality fits more naturally in the OpenMM conversion script rather than in here.  At least to me.